Frends.GooglePubSub.Publish/Frends.GooglePubSub.Publish/Definitions/Message.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Google.Cloud.PubSub.V1;
+using Google.Protobuf;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Frends.GooglePubSub.Publish.Definitions
+{
+    /// <summary>
+    /// Message attribute.
+    /// </summary>
+    public class MessageAttribute
+    {
+        /// <summary>
+        /// Default ctor.
+        /// </summary>
+        public MessageAttribute()
```

Comment:
This constructor is not used, so it can be removed.

Reply:
I do believe that Frends requires a default constructor for instantiating this class.

Reply:
No, it doesn't. If that would be the case, then we would have to provide empty constructors for all parameter classes.

Reply:
Not exactly. By default if you do not define any constructors yourself in a C# class then that class automatically gets a public default constructor (and this constructor is then used e.g. by Frends to create the parameters etc).

However IF you define at least some constructor explicitly (as with this class I created a ctor with parameters) then the default ctor is NOT generated for you, thus making it impossible for the class to be instantiated (without reflection usage). So the reason for this class to have a default ctor manually defined is because there is another ctor with params.